src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+            with special rights or restrictions.
+            <span class="read-more-btn" @click="readMoreFlag = true">
+              <span v-if="!readMoreFlag">Read more...</span>
+            </span>
+            <div v-if="readMoreFlag">
+              <div class="read-more-line">
+                If the corporation's share structure contains a class or series with special rights or restrictions, you
+                need to create a unique set of Articles for the company and outline these special rights or restrictions
+                in the Articles.
+              </div>
+              <div class="read-more-line">
+                We recommend seeking professional assistance from a lawyer or accountant to help you prepare your
+                Incorporation Agreement and Articles.
+                <span class="read-more-btn" @click="readMoreFlag = false">
+                  <span>Read less...</span>
+                </span>
+              </div>
+            </div>
+          </div>
+          <div class="preview-download-container">
+            <div>
+              <img src="@/assets/images/BCRegistries_Sample_IncoporationAgreement_x2.png" class="preview-image" />
+            </div>
+            <div class="download-link-container">
+              <span>
+                <v-icon color="blue">mdi-file-pdf-outline</v-icon>
+                <a href="/files/benefit_company__corporation_agreement.pdf" download>
+                  Download the Sample Incorporation Agreement and Benefit Company Articles
+                </a>
+              </span>
+            </div>
+          </div>
+        </v-card>
+      </div>
+    </section>
+    <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>3. Confirm Incorporation Agreement and Article Completion</h2>
+      </header>
+      <AgreementType />
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { AgreementType } from '@/components/IncorporationAgreement'
+
+@Component({
+  components: {
+    AgreementType
+  }
+})
+export default class IncorporationAgreement extends Vue {
+  private helpToggle: boolean = false
+  private readMoreFlag: boolean = false
+
+  private get documentURL (): string {
+    return 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/' +
+    'business-management/permits-licences-and-registration/registries-forms/' +
+    'benefit_company__corporation_agreement.pdf'
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+ul {
+  list-style-type: disc;
+}
+
+li {
+  padding-top:0.25rem
+}
+
+.articles-statements{
+  margin-left: 1.3rem;
+}
+
+.articles-statements-footer {
+  margin-left: 1.8rem;
+  margin-top: 1.2rem
+}
+
+p{
+  padding-top: 0.5rem;
+}
+
+a {
+  text-decoration: none;
+  color: #1976d2;
```

Comment:
This colour value is used in a number of places. I suggest adding a variable for it to theme.scss and replacing occurrences of it with its variable (wherever possible).

Reply:
This is Vuetify's primary button colour. If you can't find a name for it, you could call it "primary-blue" 😁

Reply:
It's named "blue darken-2".

Reply:
Ok. will do